### ISSUE TITLE
React logo not accessible

### ISSUE THREAD

**Issue by xydlebug** [xydlebug/X-ttt#2]
The React logo requires `alt=""` and `role="presentation"`

See https://dequeuniversity.com/rules/axe/4.8/image-alt?application=AxeChrome
